src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -492,15 +518,65 @@ private async Task ExecuteClusterTransactionTask()
             }
         }
 
+        struct ClusterTransactionBatch : IDisposable
+        {
+            private readonly ClusterTransactionCommand.SingleClusterDatabaseCommand[] _data;
+            public int Count = 0;
```

Comment:
private

Reply:
Was it pushed? It's still `public`

Reply:
@karmeli87 , did you reffer to `ClusterTransactionBatch` or `Count`?